lib/runtime-abi/Cargo.toml

```diff
@@ -13,7 +13,7 @@ wasmer-runtime-core = { path = "../runtime-core" }
 hashbrown = "0.1"
 failure = "0.1"
 tar = "0.4"
-wasmparser = "0.30.0"
+wasmparser = "0.29.2"
```

Comment:
Is this intended?

Reply:
Yes, it's to match the current version cranelift is using, compiler will complain when different versions are used together.